ci-operator/step-registry/openshift/e2e/fips/openshift-e2e-fips-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
why this not a test in openshift-test binary?

Reply:
I'm fine with that too.  There should be some way to figure out what the configured value should be for a node?  But I guess that wouldn't cover "installer ignores `fips` setting in the install-config".  Do we care about covering that portion?  We could probably cover it in `create manifests` integration testing in the installer.